environment/prod/deployment/prod/.env.prod.secrets

```diff
@@ -8,6 +8,8 @@ DB_ROOT_PASSWORD=ENC[AES256_GCM,data:F8CSHYLnv+sbhzQEEWS/zXkfBeRYE+MtPxvensa1,iv
 MAIL_USERNAME=ENC[AES256_GCM,data:gPwuz/esHsvNqpofGIE7tB9urA==,iv:Opr1LaRT1zHA60KIUE5gvvVRP/4eyJca4D4vDTCpV/k=,tag:uk49S3jiX+qwz3XGnCdwcw==,type:str]
 MAIL_PASSWORD=ENC[AES256_GCM,data:1Q+azNrjsJTGi2yHm3gao/t/yRRg,iv:i4A/ajRxedmCYJuq47uDA7KKBD0RjwBm63SWBjvHrNY=,tag:0Q8ELFTKzk751BZVVShxsw==,type:str]
 ANALYTICS_URL=ENC[AES256_GCM,data:MeXp30hd378q5DP8cYWi6RHQ0fhwIsI80rgN6BIOz3ymV/l5qNtj/pizpzIXYZAf86J6MT9vYr/u,iv:rgFnI7UVtsN8qs7qSFDbOHWF6tMiHILYqdb66oyi/tg=,tag:QJWO+keao0r/QVLixVdONA==,type:str]
+RECAPTCHAV3_SITEKEY=
+RECAPTCHAV3_SECRET=-
```

Comment:
Will you set it?

Reply:
I don't have access to this google project

Reply:
So please ping responsible person and set it up.